oaknational/Oak-Web-Application
# PUPIL-348: Serve quiz images from new Cloudinary distribution

## Description

Music year: [1971](https://www.youtube.com/watch?v=MV0F_XiR48Q)

With the new pupil pages we made a change to serve quiz images directly from Cloudinary instead of passing them through Next's image processing proxy. This was done to improve image loading which has been greatly improved. Unfortunately this had the unintended side-effect of moving requests for those images off `thenational.academy` domain. This increased the chances of a request being blocked by a school network policy and possibly incurring additional tariffs for some users.

This change moves those requests back under a `thenational.academy` origin mitigating those issues.

## How to test

1. Go to https://deploy-preview-2269--oak-web-application.netlify.thenational.academy/pupils/programmes/english-primary-ks2/units/developing-reading-preferences-in-year-5/lessons/developing-reading-preferences-in-year-5-through-appreciation-of-characters
2. Take the quiz
3. Inspect each image and you should see that it is served from a Cloudinary distribution under `thenational.academy` and it should load no differently than before.
4. Go to https://deploy-preview-2269--oak-web-application.netlify.thenational.academy/pupils/programmes/science-secondary-ks3/units/moving-by-force/lessons/reading-distance-time-graphs
5. Take the starter quiz
6. Inspect the images in the answers and they should also be served from a Cloudinary distribution under `thenational.academy`

<img width="1767" alt="Screenshot 2024-02-20 at 12 50 06" src="https://github.com/oaknational/Oak-Web-Application/assets/122096/aefabb76-482f-4e3f-95fb-0a22b0a448d3">
<img width="1792" alt="Screenshot 2024-02-20 at 12 51 04" src="https://github.com/oaknational/Oak-Web-Application/assets/122096/03b84f59-3605-4514-a87f-07d3526cc4fa">


## Checklist

- [x] Added new env vars to development and production configs.
